### PR TITLE
Fix for current cloud jira version

### PR DESCRIPTION
**Pull request by arielscarpinelli**
I don't know if it makes sense to keep support for what was implemented before, but it was not working with current cloud version

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -234,9 +234,9 @@
 		},
 
 		sumCols: function () {
-			const columnIds = this.$headerCols.map(($column) => {
+			const columnIds = this.$headerCols.map(($column, idx) => {
 				if (this.isBoardTypeNextGen) {
-					return $column.dataset.rbdDragHandleDraggableId;
+					return idx;
```
Comment (arielscarpinelli) [inline index.js:239]: so there is no information in the html about which column it is anymore, but just walking them in order and using their position as "id" works fine.

File: index.js [hunk 2]
```diff
@@ -336,7 +336,7 @@
 						colSum
 					);
 
-					$targetCol.children[0]?.appendChild($colSum);
+					$targetCol.children[0]?.prepend($colSum);
```
Comment (arielscarpinelli) [inline index.js:339]: I think this always has been a bug; as the stuff that should float must appear first in the HTML.

File: index.js [hunk 3]
```diff
@@ -76,11 +76,11 @@
 				observeRefreshSelector = ".ghx-column-header-group";
 			} else if (this.isBoardTypeNextGen) {
 				headerColsSelector =
-					'[data-test-id="platform-board-kit.common.ui.column-header.header.column-header-container"]';
+					'[data-testid="platform-board-kit.common.ui.column-header.header.column-header-container"]';
 				laneColsSelector =
 					'[data-component-selector="platform-board-kit.ui.column.draggable-column"]';
 				estimateSelector =
-					'[data-testid="software-board.common.fields.estimate-field.static.estimate-wrapper"]';
+					'[data-testid="platform-card.common.ui.estimate.badge"]';
```
Comment (filip-savic) [inline index.js]: @arielscarpinelli Hi, first of all thanks for the PR :)

i tested this out the Jira cloud version, and the deleted one still seems to be working there. Maybe the best solution would be to just include both:

```
estimateSelector =
					'[data-testid="platform-card.common.ui.estimate.badge"], [data-testid="software-board.common.fields.estimate-field.static.estimate-wrapper"]';
```

Comment (arielscarpinelli) [reply index.js]:
```suggestion
					'[data-testid="platform-card.common.ui.estimate.badge"], [data-testid="software-board.common.fields.estimate-field.static.estimate-wrapper"]';
```

**Pull request merged**